status/status.go

```diff
@@ -54,7 +54,7 @@ func StateForContainerStatus(status *corev1.ContainerStatus) (State, *int32) {
 	if terminateState := status.State.Terminated; terminateState != nil {
 		var state State = Errored
 
-		if terminateState.ExitCode == 0 {
+		if terminateState.ExitCode == 0 || terminateState.ExitCode == 2 {
```

Comment:
This seems odd. Why is this necessary?

Reply:
As discussed, I will have a separate PR to address this.